01-shapefile-attributes.Rmd

```diff
@@ -1,15 +1,18 @@
 ---
 layout: post
-title: "Lesson 01: Work With Shapefile Attributes in R"
+title: "Lesson 01: Shapefile Metadata & Attributes in R"
```

Comment:
If you think some of the titles are too jargony - let's open a n issue. i'm open to using different words. but i wasn't sure if people knew either to be honest. shapefile vs vector - i think generally people have seen a shapefile if they've heard of one of those?